### DISCUSSION TITLE
Other TMS coil model

### DISCUSSION THREAD

**Discussion by yvalter** [simnibs/simnibs#217] [8. Simulation workflows] [answered]
Which coil model should I use to most accurately simulate the e-field of Neurosoft's CloudTMS system?

**Comment by Kstoffer** [comment]
In order to accurately model that coil, the exact wire geometry would have to be known. We currently do not have that information available.

**Comment by TorgeW** [comment, marked as answer]
If you do have this information you can now create your own coil files.

We released a pre-release which implements our new coil format.
https://github.com/simnibs/simnibs/releases/tag/dev

Examples on how to use it can be found here:
https://github.com/simnibs/simnibs/tree/master/simnibs/examples/coils

**Comment by FrancescoLomi** [comment]
Hi everybody,

I'm trying to create a figure of 8 coil for my stimulator but I encounter an error when i run the script "create_simple_figure_of_8_coil.py"(https://github.com/simnibs/simnibs/blob/master/simnibs/examples/coils/create_simple_figure_of_8_coil.py). Below there is the screenshot of the error from python interface.
It seems an error related to the scipy package, I've tried to reinstall both scipy and numpy, but nothing seems to change. 
How could I fix this?
Thank you in advance
Best
Francesco

![Screenshot 2024-01-26 133018](https://github.com/simnibs/simnibs/assets/118944911/a6242f25-9747-4451-80b2-edeb70b48630)

**Reply by TorgeW** [reply]
Try to reinstall simnibs.
Otherwise you should reach out to the scipy support.
